source/User_Guide/Marketing_Campaigns/lists.md

```diff
@@ -99,46 +99,57 @@ The fields `Date Added` and `Last Updated` are system generated. `Date Added` re
 You can create up to 400 segments.
 {% endinfo %}
 
-To set up a segment, click the “add list or Segment” button from [Contacts]({{site.marketing_campaigns_url}}/contacts).
+*To create a Segment:*
 
-![]({{root_url}}/images/lists_5.png "Delete a List")
+1. To set up a segment, click the **Add List or Segment** button from [Contacts]({{site.marketing_campaigns_url}}/contacts).
```

Comment:
1. Navigate to **Marketing** and then select **Contacts**.
1. Click **Add List or Segment**.